profiles/student/axk4545.yml

```diff
@@ -0,0 +1,12 @@
+name: Aidan Kahrs
+website: https://axk
+blog: https://axk4545.me
+email: public@axk4545.me
+major: Networking and Systems Administration
+graduation: 2018
+handle: axk4545
+forges:
+  GitHub: https://github.com/axk4545
+  GitLab: https://gitlab.com/axk4545/
+bio: Sys admin with a passion for Linux and the FOSS community. Linux user since approximately 2012. Currently running Fedora Linux. 
```

Comment:
It looks like there is trailing whitespace at the end of this line – could you erase the whitespace?